main.go

```diff
@@ -72,76 +65,37 @@ import (
 )
 
 var (
-	ErrInvalidLogLevel = errors.New("--log-level has to be one of debug, info, warn, error, fatal")
-
-	certDir            string
-	logLevel           string
-	scheme             = runtime.NewScheme()
-	setupLog           = ctrl.Log.WithName("setup")
-	telemetryNamespace string
-	enableWebhook      bool
-
-	maxLogPipelines    int
-	maxTracePipelines  int
-	maxMetricPipelines int
-
-	traceGatewayImage                string
-	traceGatewayPriorityClass        string
-	traceGatewayCPULimit             string
-	traceGatewayDynamicCPULimit      string
-	traceGatewayMemoryLimit          string
-	traceGatewayDynamicMemoryLimit   string
-	traceGatewayCPURequest           string
-	traceGatewayDynamicCPURequest    string
-	traceGatewayMemoryRequest        string
-	traceGatewayDynamicMemoryRequest string
-
-	fluentBitDeniedFilterPlugins string
-	fluentBitDeniedOutputPlugins string
-	fluentBitMemoryBufferLimit   string
-	fluentBitFsBufferLimit       string
-	fluentBitCPULimit            string
-	fluentBitMemoryLimit         string
-	fluentBitCPURequest          string
-	fluentBitMemoryRequest       string
-	fluentBitImage               string
-	fluentBitExporterImage       string
-	fluentBitPriorityClassName   string
-
-	metricGatewayImage                string
-	metricGatewayPriorityClass        string
-	metricGatewayCPULimit             string
-	metricGatewayDynamicCPULimit      string
-	metricGatewayMemoryLimit          string
-	metricGatewayDynamicMemoryLimit   string
-	metricGatewayCPURequest           string
-	metricGatewayDynamicCPURequest    string
-	metricGatewayMemoryRequest        string
-	metricGatewayDynamicMemoryRequest string
-
-	selfMonitorImage         string
-	selfMonitorPriorityClass string
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup").WithValues("version", version)
```

Comment:
logging the version with every logline of this logger seem a bit excessive

Reply:
Before, the version was added as a key to a random log entry. I can also log it once somewhere at the top of the `run` function:
```go
logger.Info("Starting Telemetry Manager", "version", version)
```

Reply:
Done